src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -241,6 +241,38 @@ std::string parseSubscription(ConnectionInfo* ciP, SubscriptionUpdate* subsP, bo
     subsP->throttling = 0;  // Default value if not provided at creation => no throttling
   }
 
+  // MaxFailsLimit
+  Opt<int64_t> maxFailsLimitOpt = getInt64Opt(document, "maxFailsLimit");
+  if (!maxFailsLimitOpt.ok())
+  {
+    return badInput(ciP, maxFailsLimitOpt.error);
+  }
+  else if (maxFailsLimitOpt.given)
+  {
+    subsP->maxFailsLimitProvided = true;
+    subsP->maxFailsLimit = maxFailsLimitOpt.value;
+  }
+  else if (!update)  // maxFailsLimit was not set and it is not update
+  {
+    subsP->maxFailsLimit = 0;  // Default value if not provided at creation => no maxFailsLimit
+  }
+
+  // FailsCounter
+  Opt<int64_t> failsCounterOpt = getInt64Opt(document, "failsCounter");
+  if (!failsCounterOpt.ok())
+  {
+    return badInput(ciP, failsCounterOpt.error);
+  }
+  else if (failsCounterOpt.given)
+  {
+    subsP->failsCounterProvided = true;
+    subsP->failsCounter = failsCounterOpt.value;
+  }
+  else if (!update)
+  {
+    subsP->failsCounter = 0;
+  }
```

Comment:
failsCounter is not a field to be provided by the client in subscription requests (note the client only provided the maxFailsLimit and faildCounter in a calculated field by CB). Thus no need of parsing it and this block of code should be removed.

Reply:
Removed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de